### PR TITLE
Adds a check for RemoteTech Antennae.

### PR DESCRIPTION
**Pull request by ABZB**
It will only check for the RT module if it looks for and fails to find the stock antenna module.

### CODE REVIEW

File: Helpers/VesselHelper.cs [hunk 1]
```diff
@@ -74,6 +74,12 @@ public static class VesselHelper {
           if (Module.moduleValues.GetValue ("name") == module) {
             return true;
           }
+		  //if the module being searched for is ModuleDataTransmitter (stock antenna), and that module was not found, now look for ModuleRTAntenna, the RemoteTech Antenna Module.
```
Comment (micah-williamson) [inline Helpers/VesselHelper.cs:77]: Gotcha. Let me clean this up a bit by adding a aliases so we don't end up with hundreds of edge cases and I'll merge this in. Thanks!

Comment (ABZB) [inline Helpers/VesselHelper.cs:77]: no problem.

Comment (ABZB) [inline Helpers/VesselHelper.cs:77]: if you are making a full alias list, for the ModuleSAS in the satellite
check (which I am guessing is to check for a probe part), RT gives exactly
every probe the module "ModuleSPU" - which is nice, as there are some
probes which lack ModuleSAS.

**Comment by micah-williamson** [comment]
Added `data/modulealiases.settings` here: https://github.com/iamchairs/StateFunding/pull/5

**Pull request closed without merging**